src/main/views/serverDropdownView.ts

```diff
@@ -75,7 +75,7 @@ export class ServerDropdownView {
 
     private init = () => {
         log.info('init');
-        const preload = getLocalPreload('desktopAPI.js');
+        const preload = getLocalPreload('internalAPI.js');
```

Comment:
I didn't realize all the BrowserViews that make up the desktop app also needed this preload script. I guess it makes sense since it's also how they access the desktop API